main.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'lib/count_words'
+
+counter = Count_Words.new
+
+File.open('speech.txt','r') do |file|
+    file.each do |line|
+        counter.count_words(line)
+    end
+    counter.order_book_by_count
+    counter.render_book
+end     
```

Comment:
I haven't seen this strategy before. Typically folks will give all the file content to the object and then ask for a report. Something like:

```ruby
corpus = File.read('speech.txt')
counter = Count_Words.new(corpus)
puts counter.words
```
But I don't mean to imply there's anything wrong with your approach. Reading the file one line at a time is great for really big files. 

If you feel like practicing more, you could try switching approaches. This is pretty awesome as-is, though. Let me know if you would like any additional guidance.